python/foxglove-sdk/src/websocket_server.rs

```diff
@@ -356,6 +360,12 @@ pub fn start_server(
         server = server.services(services.into_iter().map(PyService::into));
     }
 
+    if let Some(asset_handler) = asset_handler {
+        server = server.fetch_asset_handler(Box::new(AssetServer {
```

Comment:
Naming wise, the "AssetServer" name makes me think this is a standalone thing and not something I would necessarily expect to be passed in as an "asset handler"

could we call it something with AssetHandler in the name (I see AssetHandler is taken though) – PyAssetHandler? CallbackAssetHandler?

I guess using @eloff's suggestion of a closure would also resolve this concern because it would no longer need a name :)

Reply:
Renamed to `CallbackAssetHandler` for now; I'll see if I can figure out conversions when I revisit the examples. This name isn't exposed to the python side, so I'd like to avoid the `Py` prefix.